docs-src/modules/ROOT/pages/loading-data.adoc

```diff
@@ -42,10 +42,7 @@ CLI::
 +
 [source,bash]
 ----
-kaskada-cli load \
-    --table Purchase \
-    --file-path file:///path/to/file.parquet \
-    --file-type parquet
+kaskada-cli table load Purchase file:///path/to/file.parquet
```

Comment:
Can we use the same notes about file type here as are used for the other sections?